Source/PeleLMDiagnostics.cpp

```diff
@@ -32,6 +32,15 @@ PeleLM::createDiagnostics()
         bool itexists = derive_lst.canDerive(v) || isStateVariable(v) || isReactVariable(v);
         if (!itexists) {
             Abort("Field "+v+" is not available");
+        } else {
+            if (derive_lst.canDerive(v)) {
+                const PeleLMDeriveRec* rec = derive_lst.get(v);
+                if (rec->numDerive() > 1) {
```

Comment:
This will capture when people request the individual derive components as well. The condition `rec->variableComp(v) < 0` should work to throw an error for the full multi-component derive but not the individual components.

Reply:
Indeed ! Good catch !